validation/validation.go

```diff
@@ -16,10 +16,16 @@ func Validate(cfg *config.Local, tracker issuetracker.IssueTracker) []error {
 		errs = append(errs, err)
 	}
 
-	if err := validateAuthOfflineURL(cfg); err != nil {
-		errs = append(errs, err)
+	if cfg.Auth.Type == config.AuthTypeOffline{
+		if err := validateAuthOfflineURL(cfg); err != nil {
+			errs = append(errs, err)
+		}
+		if err := validateAuthOfflineUrlIsSet(cfg); err != nil {
+			errs = append(errs,err)
+		}
```

Comment:
I'd prefer if we first check that the auth offline url is set and use an `else if` to make the two checks mutually exclusive - if the offline url is not set, there is no point in checking if it's valid.

In example:
```go
if err := validateAuthOfflineUrlIsSet(cfg); err != nil {
   ...
} else if err := validateAuthOfflineURL(cfg); err != nil {
   ...
}
```

Reply:
Good point! change is now implemented.

Reply:
One final thing - the checks are still reversed.

It should be `validateAuthOfflineURLIsSet` first and then `validateAuthOfflineURL`. After this, we're good to go.